organisaatio-ui/src/components/Sivut/Ryhmat/Muokkaus/MuokkausLomake.test.tsx

```diff
@@ -82,41 +92,41 @@ describe('MuokkausLomake', () => {
 
     describe('Buttons', () => {
         describe('Passivoi button', () => {
-            it('clicks', () => {
-                const wrapper = shallow(<MuokkausLomake {...testProps as MuokkausLomakeProps} />);
+            it('Calls its onClick handler when clicked', () => {
```

Comment:
I see slight frustration here. the point is not to test whether onClick works as documented but that correct action happens as expected. This could be reflected in test description: 'handlePassive in invoked when passivoi is clicked'. This is good test since it's really easy to make human error. for example this test contains error; wrong method is being verified around the simulated click